fenix/app/src/main/java/org/mozilla/fenix/IntentReceiverActivity.kt

```diff
@@ -73,6 +73,12 @@ class IntentReceiverActivity : Activity() {
 
         addReferrerInformation(intent)
 
+        if (intent.type == "application/pdf") {
```

Comment:
Should we move `application/pdf` to companion object as content?

Reply:
I forgot that, good catch, thanks! It's fixed now